site/content/en/docs/faq/_index.md

```diff
@@ -92,3 +92,15 @@ Yes! If you prefer not having emoji in your minikube output 😔 , just set the
 MINIKUBE_IN_STYLE=0 minikube start
 
 ```
+
+## How to access minikube cluster from on a remote network ?
+
+minikube is primary goal is to quickly sets up a local Kubernetes clusters, and we strongly discourge from using minikube in production or to listen on remote traffic. therefore by design minikube networking only listens on local network.
```

Comment:
minikube's primary goal is to quickly set up local Kubernetes clusters, and therefore we strongly discourage using minikube in production or for listening to remote traffic. By design, minikube is meant to only listen on the local network.